src/util/XournalType.h

```diff
@@ -26,7 +26,4 @@
 // Include string and vector everywhere
 #include <cstring>  // Todo: Use std::string_view if possible
 #include <string>
-#include <vector>
-
-using std::string;
-using std::vector;
+#include <vector>
```

Comment:
Remove this header completely.

Reply:
Done. I suggest to wait for  #2339 to get merged.

Reply:
Yes, this is a very invasive change, breaking the code of nearly every PR.